number_theory/sieve_unittest.cpp

```diff
@@ -0,0 +1,62 @@
+#include <stddef.h>
+
+#include <limits>
+#include <stdexcept>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+#include "number_theory/sieve.h"
+
+namespace tql {
+namespace number_theory {
```

Comment:
warning: nested namespaces can be concatenated [modernize-concat-nested-namespaces]

```suggestion
namespace tql::number_theory {
```

number_theory/sieve_unittest.cpp:60:
```diff
- }  // namespace number_theory
- }  // namespace tql
+ }  // namespace tql
```